druid-shell/src/platform/windows/window.rs

```diff
@@ -373,9 +392,38 @@ impl WndState {
     // Renders but does not present.
     fn render(&mut self, d2d: &D2DFactory, dw: &DwriteFactory, invalid: &Region) {
         let rt = self.render_target.as_mut().unwrap();
+
         rt.begin_draw();
         {
+            // Piet is missing alpha blending setting, so we have to call
+            // ID2D1DeviceContext::SetPrimitiveBlend() manually to clear just
+            // the required pixels
+            let dc_for_transparency: Option<&ComPtr<ID2D1DeviceContext>> = if self.transparent {
```

Comment:
As of 1.50 there is the very helpful [`bool::then`](http://doc.rust-lang.org/1.50.0/std/primitive.bool.html#method.then) for this sort of situation...